gmaildump/gmailhistory.py

```diff
@@ -158,33 +183,39 @@ def watch_gmail(self):
         >>> obj.gmail.users().watch().execute = Mock(obj.gmail.users().watch().execute, return_value=api_doc)
         >>> obj.watch_gmail()
         {'expiration': 1526901631234, 'historyId': 1234}
+
         '''
         self.log.debug('watch_gmail')
 
         request = {
             'labelIds': self.LABELIDS,
-            'topicName': '%s' % (self.topic)
+            'topicName': '{}'.format(self.topic)
         }
 
         hstry_id = self.gmail.users().watch(userId='me', body=request).execute()
+
         self.log.info('Gmail_watch_id :', hstryid=hstry_id)
 
         return hstry_id
 
     def send_msgs_to_target(self, target, msg):
```

Comment:
Just by reading the variable name, target is some person, whom you want ti send the message you received??
The function name name and docstring has to be in sync, if this looks sensible to you, please modify the variable names inside the fucntion declaration also.